### DISCUSSION TITLE
Won't work on Wyze Cam v3

### DISCUSSION THREAD

**Discussion by nickleman** [gtxaspec/wz_mini_hacks#604] [Q&A] [answered]
I had successfully reflashed the camera back to 4.36.9.139 and re-added it to my Wyze app.

I have tried 2 different SD cards and multiple reformatings as FAT 32 and even a repartitioning, but yet I can't get this to work.

After I put the SD Card in the camera and plug it back in, all that happens is the led blinks on 2 seconds and off seconds, and repeats. I've even waited 10 minutes and still the same blinking and never comes on the wireless.

I copied the files in SD_ROOT to the card so it has the one file and the one directory (with everything under it).

**Comment by brentfraser** [comment]
Have a look at #31

**Reply by nickleman** [reply]
Thanks. I had seen that. I had originally done mine on Linux, then tried Windows, then went back to Linux. I'm using a 32 GB card.

**Comment by virmaior** [comment]
1. have you previously set up the camera with the wyze app?
2. what are the settings in your wz_mini.conf ?
3. what is in the log file on the SD card? `/wz_mini/log/wz_init.log` ?


The phenomenon you're showing sounds like an issue with the networking part of your camera.

**Comment by nickleman** [comment]
> 1. have you previously set up the camera with the wyze app?
Yes, it was running for several years, then I reverted it to the compatible FW version (4.36.9.139 ), re-added it to the Wyze app and it has been working there for several days prior to and after trying this.

> 2. what are the settings in your wz_mini.conf ?
The only thing I changed from the default was to add my SSH auth key to the authorized keys file.

> 3. what is in the log file on the SD card? `/wz_mini/log/wz_init.log` ?
I haven't checked it, I didn't think it had even booted from the file. I'll check that later today.
 
> The phenomenon you're showing sounds like an issue with the networking part of your camera.
There were no TTS statements from the camera at all so far. I think I saw others that have mentioned they heard statements of progress. I'll double check, but I thought the config file already had the two standard wireless modules enabled.

Thanks for the help and the ideas.

EDIT: I just checked and there is no log file.

**Comment by virmaior** [comment]
if there's no `/wz_mini/wz_init.log` then wz_mini_hacks isn't running  at all.
no chance you left the downgrade firmware in the root at the same time?

**Reply by nickleman** [reply]
I did not. it just has the two items, the lib and the wz_mini folder. Do you know what the sha256 should be for the call back library? just to make sure it is on the flash card correctly?

**Reply by virmaior** [reply]
a failure to boot *and* no log on the SD card is a weird combination because
1. if wz_mini_hack is not running, then it should be booting identically to no SD card
2. if wz_mini_hack is running, there should be log files

no idea for the sha256 on that. You're referring to the `factory_t31_ZMC6tiIDQN` file? (still not something I know).

Basically that file in the root must be on a FAT partition (not exFAT) and the camera's bootloader uses it before checking anything else. but it is possible that wyze changed the bootloader at some point.

**Comment by Bob-AOL** [comment]
Start from scratch with a fresh SD card and fresh 4.36.9.139 download from Wyze and re-flash.

It has to be something simple within those basic aspects..., or the camera is scrambled somehow. Maybe a card reader going south. It's happened to me and had me chasing my tail for days.

And just to be clear, the only items on the sd card should be the t31 ID and the wz_mini folder. Nothing else.

**Reply by nickleman** [reply]
OK, I will try it again. I'll reflash the 4.36.9.139 and then try it again with the new sd card. It'll be a few days before I can get the time to do it again. 

Thanks.

**Reply by nickleman** [reply]
Unfortunately, it still didn't work. I tried going back to ...131 instead of 139 as well. The rollbacks worked, but not the wz_mini_hacks. Oh, well, I guess I'll just stick with straight Wyze, for now. When I have some more time, I'll look into this again.

**Reply by nickleman** [reply, marked as answer]
Actually, I got a little time last night and looked at my git repo clone and even though I run on linux my git clone converted all text file line endings to windows line endings. So I downloaded a zip of the repo instead and that worked. I'll have to look into why a linux to linux file cloning in git converted line endings, but that's for another day.

Thanks for the help!
